test/extended/fixtures/ldap/ldapserver-buildconfig.json

```diff
@@ -17,6 +17,7 @@
       },
       "contextDir": "images/openldap"
     },
+    "runPolicy": "Serial",
```

Comment:
added a test to make sure runPolicy is defaulting to Serial, reverted the change to this json file